x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/create_package_policy_page/multi_page_layout/components/page_steps/add_integration.tsx

```diff
@@ -223,7 +245,7 @@ export const AddIntegrationPageStep: React.FC<MultiPageStepLayoutProps> = (props
       <NotObscuredByBottomBar />
       <CreatePackagePolicyBottomBar
         cancelClickHandler={isManaged ? onBack : () => setIsManaged(true)}
-        onNext={onSubmit}
+        onNext={() => onSubmit()}
```

Comment:
(nit) Could be shortenened to `onNext={onSubmit}` since there's no args to deal with here